src/test/regress/bin/normalize.sed

```diff
@@ -13,11 +13,20 @@ s/shard [0-9]+/shard xxxxx/g
 
 # In foreign_key_to_reference_table, normalize shard table names, etc in
 # the generated plan
-s/"(fkey_ref_|referenced_table_|referencing_table_)[0-9]+"/"\1xxxxxxx"/g
+s/"(foreign_key_2_|fkey_ref_to_dist_|fkey_ref_)[0-9]+"/"\1xxxxxxx"/g
```

Comment:
I think we can also update `with_modifying.out`, there is only a minor error message update is missing.